redi-connect-front/src/components/organisms/PreFooter.scss

```diff
@@ -0,0 +1,38 @@
+@import "~bulma/sass/utilities/_all";
+@import "../../styles/_variables.scss";
+
+.pre-footer {
+  &-container {
+    margin: 1.5rem auto;
+
+    @include mobile() {
+      margin-bottom: -2.5rem;
+    }
+  }
+
+  &-heading.title {
+    margin-bottom: 1.5rem;
```

Comment:
we should have this standardized.... have you checked the standard spacings? 
I will become a problem if we set non standard spacings at the end...